src/main/groovy/edu/oregonstate/mist/mealcard/resources/MealPlanResource.groovy

```diff
@@ -0,0 +1,61 @@
+package edu.oregonstate.mist.mealcard.resources
+
+import com.codahale.metrics.annotation.Timed
+import edu.oregonstate.mist.api.Resource
+import edu.oregonstate.mist.api.jsonapi.ResourceObject
+import edu.oregonstate.mist.api.jsonapi.ResultObject
+import edu.oregonstate.mist.mealcard.db.MealPlanDAO
+
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.core.Context
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.UriBuilder
+import javax.ws.rs.core.UriInfo
+import javax.ws.rs.core.Response
+
+@Path("diningbalances")
+@Produces(MediaType.APPLICATION_JSON)
+class MealPlanResource extends Resource {
```

Comment:
Let's add a couple of annotations here that most of our recent APIs are using: 
@PermitAll - allows us to specify that a user authentication is required for the endpoint.
@TypeChecked - tells the compiler that we want it to check types